cpp/ps0004/cpp_ps0004_solution.cpp

```diff
@@ -11,29 +12,26 @@ int main()
     int point = num;
     char ascii = num;

```

Comment:
- [x] declared a variable to store the user input
- [x] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input